### DISCUSSION TITLE
Plugin scan incomplete

### DISCUSSION THREAD

**Discussion by DBall1962** [apohl79/audiogridder#423] [Q&A]
Hi All,
New to this platform and I really like what I see so far. I'm having an issue with the server software not picking up on all the plugins. Specifically the problem seems to be with audio unit plugins for Logic Pro X. When I check the library\audio\plugins\components folder, everything is there but the server doesn't catch it in the scan (or wipe cache and rescan). My system is simple, I use a MacBook pro running Mojave to run the server and use a thunderbolt bridge connection to a mac mini also running Mojave. Any suggestions would be helpful. Thx in advance.

**Comment by CaBuffalo** [comment]
In the plugin filter of AG in the DAW, there is a "Disable server filter"
option. Check it and see if it works.

On Sat, Mar 6, 2021 at 6:55 PM DBall1962 <notifications@github.com> wrote:

> Hi All,
> New to this platform and I really like what I see so far. I'm having an
> issue with the server software not picking up on all the plugins.
> Specifically the problem seems to be with audio unit plugins for Logic Pro
> X. When I check the library\audio\plugins\components folder, everything is
> there but the server doesn't catch it in the scan (or wipe cache and
> rescan). My system is simple, I use a MacBook pro running Mojave to run the
> server and use a thunderbolt bridge connection to a mac mini also running
> Mojave. Any suggestions would be helpful. Thx in advance.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/423>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6EHP3LYC7WFMYHZZVTTCLTJRANCNFSM4YXL7KBA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Comment by DBall1962** [comment]
Hi and thanks for the feedback. The issue I'm having is on the server side not the plugin side. I did do what you recommended however if the plugin is not on the server side it wont be seen on the plugin (client) side. Specifically I'm trying to get the Slate Digital suite to be recognized on the server side. Any other thoughts are more than welcome. Thanks again.

**Comment by apohl79** [comment]
Do you see the missing plugins when you run "auval -al" in the terminal app? And do you see the missing plugins in a DAW if you have one on the server?

**Reply by DBall1962** [reply]
Hi and thank you for your guidance. I attempted to run the auval -al command in terminal and I get an invalid argument returned so I may not be doing that correctly. Please advise if there is special syntax needed to run that properly. On the server computer (MacBook Pro) I can run Logic Pro X 10.5.1 and all the Slate Digital plugins are there and they work great. When I attempt to use the Audiogridder plugin over the thunderbolt bridge for the AU plugins, allot of the plugins are not there. The same missing plugins are there under the VST or VST3 format. I appears that the AU version is the issue. Please advise on any other thoughts. Thanks again.


![image](https://user-images.githubusercontent.com/80184847/110510681-72f34d80-80d1-11eb-87f8-e4841bf29888.png)
